### PR TITLE
Get filter box viz working for explore v2

### PR DESCRIPTION
**Pull request by vera-liu**
Done: 
- Added containerId and getFilters

Todo:
- get filters working in new formdata and control panel

After:
screen shot 2016-10-15 at 7 10 26 pm

@ascott

### CODE REVIEW

**Comment by mistercrunch** [comment]
We often point Airflow committers to this article bellow. Please take the time to read it as it's nice to have good commit messages (though no need to overdo it!), that makes for easier PR/project management, a good changelog and can even provide context while dealing with merge conflicts.
http://chris.beams.io/posts/git-commit/

File: caravel/assets/javascripts/explorev2/components/ChartContainer.jsx [hunk 1]
```diff
@@ -42,6 +44,16 @@ class ChartContainer extends React.Component {
 
       selector: `#${this.props.sliceContainerId}`,
 
+      setFilter: () => {
+        // set filter according to data in store
```
Comment (vera-liu) [inline caravel/assets/javascripts/explorev2/components/ChartContainer.jsx]: https://github.com/airbnb/caravel/issues/1425

**Comment by ascott** [comment]
LGTM

**Comment by ascott** [comment]
looks like the tests were passing before merging... what do you think happened? should the revert be merged?

**Comment by vera-liu** [comment]
Seems that master was broken when I merged

**Pull request merged**